airbyte-protocol/protocol-models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml

```diff
@@ -430,6 +429,7 @@ definitions:
             "$ref": "#/definitions/OAuthConfigSpecification"
   OAuthConfigSpecification:
     type: object
+    additionalProperties: true
```

Comment:
I think that additional properties are allow by default ([docs](https://json-schema.org/understanding-json-schema/reference/object.html#additionalproperties)), so in practice, we should be able to remove all those lines.